### PR TITLE
Use Google Search instead of custom search

### PR DESCRIPTION
**Pull request by keanulee**
Redirects searches to Google adding `site:polymer-project.org` to the search terms. I removed the existing search code here, but we can look back to this diff if we switch to a new search API.

**Note**: PR is to the 2.x branch, the currently deployed version

Staged at https://search-dot-polymer-project.appspot.com/

Ref #2477

### CODE REVIEW

**Comment by keanulee** [comment]
@arthurevans Just realized that the 2.x branch doesn't have the latest blog post, so that's not the currently deployed version either.

**Comment by arthurevans** [comment]
The blog post just hasn't been merged yet. 2.x is the correct branch.

**Comment by arthurevans** [comment]
Ugh. Yeah, this works as a temporary fix. Back to our 1.0 site, lolz.

File: app/elements/pw-shell.html [hunk 1]
```diff
@@ -347,13 +329,6 @@
         this.async(this._resetScrollPosition);
       }
 
-      // If we have any query parameters, that means we are displaying the
```
[the remark this answers is not in the document]
Comment (keanulee) [reply app/elements/pw-shell.html]: I'm inclined to remove it and just reference this PR from the issue given that this "when" may be a while...

**Comment by merlinnot** [comment]
Last time I checked Google was the best search engine, so maybe you should actually drop this custom code and use [cse](https://cse.google.com/cse/)? I guess as a Google employees you guys can get it for free for this project ;)

**Comment by keanulee** [comment]
@merlinnot We used to use a version of CSE, but that version is now gone (you can see the XHR failing when searching with the current site). This is intended to be a stop-gap patch until we use the new thing.

**Comment by merlinnot** [comment]
Oh, I see, didn't know it's gone 😭

**Comment by arthurevans** [comment]
@merlinnot Yeah, this custom code used the v1 version of the CSE API. v2 will require some refactoring on our part.

**Pull request merged**